app/views/subscription/add.phtml

```diff
@@ -172,6 +172,27 @@
 				</div>
 			</div>
 
+			<div class="form-group">
+				<label class="group-name" for="curl_params_cookie"><?= _t('sub.feed.css_cookie') ?></label>
+				<div class="group-controls">
+					<input type="text" name="curl_params_cookie" id="curl_params_cookie" class="extend" value="" placeholder="<?= _t('gen.short.blank_to_disable') ?>" />
```

Comment:
Why do you use `curl_params_cookie` in subscription/add but `curl_params_cookiefile` in feed/update?
Shouldn't you have a single parameter to enable cookies?

Reply:
In feed/update, there already was the `curl_params_cookie` parameter which allows setting custom user-specified cookies. I added the `curl_params_cookiefile` parameter to pass the `CURLOPT_COOKIEFILE` option, which enables the libcurl cookie engine.

On the other hand, in subscription/add, the `curl_params_cookie` setting was not available, so I copied it over from feed/update for consistency, and added `curl_params_cookiefile` as well. The meaning is the same: `curl_params_cookie` for user-specified cookies, `curl_params_cookiefile` to enable the libcurl cookie engine.